src/styles/index.scss

```diff
@@ -1,3 +1,61 @@
-body {
+@import './variables';
+
+.page {
   margin: 0;
+  height: 100vh;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $days-gap;
+  width: $calendar-item-box-size * $calendar-item-columns + $days-gap * 6;
```

Comment:
The calculation for the width of the `.calendar` class might not be correct if `$calendar-item-columns` is not defined as an integer. Ensure that `$calendar-item-columns` is defined correctly and that the calculation accounts for all necessary gaps.